drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -41,15 +41,20 @@ class HmisSchema::ReferralPosting < Types::BaseObject
     field :denial_note, String
```

Comment:
I can't comment on line 39 above, but I think we should resolve denial reason as an enum. That way we can get rid of the `REFERRAL_POSTING_DENIAL_REASON_TYPES` pick list option, and just use `pickListReference: <name-of-enum>`. Whenever possible I'd prefer to use Enums as pick lists, so that we don't have to make unnecessary queries for picklists. (The frontend bundle already has all the enums, so it can generate the picklists locally).

We just got the denial reason values today, they are:
```
HMIS user error
Inability to complete intake
Does not meet eligibility criteria
No longer interested in this program
No longer experiencing homelessness
Estimated vacancy no longer available
Enrolled, but declined HMIS data entry
```

They will have integer values, but they're not created yet. Let's just use numbers 1-7 for now, since those are accepted values from the old mapping.

Let me know if that's unclear. happy to jump on a call!

Reply:
this is clear, thanks!